internal/controller/appconfigurationprovider_controller.go

```diff
@@ -51,9 +51,10 @@ type AzureAppConfigurationProviderReconciler struct {
 type ReconciliationState struct {
 	Generation                              int64
 	ConfigMapResourceVersion                *string
-	SentinelETags                           map[acpv1.Sentinel]*azcore.ETag
```

Comment:
I thought the Sentinel Etags is still required to be noted, we still need to check etags of the sentinels.

Reply:
Yes, if user set sentinels, they would be cached  in `KeyValueEtags`.